<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Messgae</class>
 <widget class="QDialog" name="Messgae">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget_title" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_ico">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>31</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>232</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>40</width>
      <height>28</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pushButton_menu_closed">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>0</y>
       <width>40</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_main" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>311</width>
     <height>115</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_ico_main">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>45</width>
         <height>45</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_content">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>10</y>
         <width>211</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>确定要删除吗？</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>281</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>58</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_ok">
          <property name="font">
           <font>
            <family>微软雅黑 Light</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>确定</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Cancle">
          <property name="font">
           <font>
            <family>微软雅黑 Light</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
